### ISSUE TITLE
Possibility to use skos:Concept for codelists

### ISSUE THREAD

**Issue by DieterDePaepe** [Informatievlaanderen/OSLO-EA-to-RDF#1]
Enumerations are currently mapped to owl:Class. It should be possible to allow mapping to skos:Concept as well. (Probably a tag in the enumeration?)

**Comment by bertvannuffelen** [comment]
Personally, I think skos:Concepts are better suited for codelists, however if one likes to do correct and complete formal reasoning they have to be integrated in owl as instances of an owl:Class.

From a perspective of data management SKOS is much better suited than OWL for controlled vocabularies. But to get a correct interplay bewteen OWL and SKOS  there have to be done some additional ontological work. This is illustrated below in the example. 

Let _:Status be a owl:Class representing statuses.
Let _:up and _down be instances of _:Status.

According to SKOS, the statements are

_:up a skos:Concept
_:up a skos:isTopConceptOf _:statusScheme.
_:up skos:prefLabel "NaarBoven"@nl.
...
_:statusScheme a skos:skos:ConceptScheme

Then in the OWL ontology there is a range restriction on a property myStatus stating that it only can have values from the Status controlled list.

using owl classes this is trivial:
myStatus rdfs:range _:Status

however expressing that the _:Status is composed of all and only the concepts from the conceptscheme _:statusScheme is much more complicated. This requires a closure statument.

**Comment by DieterDePaepe** [comment]
So you suggest always using the skos mechanism?

What are the benefits/downsides of having a closure statement? And can't it be specified with something like? (OWL is not my forte, so I might be wrong.)
```
_:Status owl:equivalentClass [
   owl:oneOf (_:up _:down)
]
```

**Comment by bertvannuffelen** [comment]
Yes.

For the closure with two items this is easy. Suppose you have 10000, and you have to manage this in  a file as a single statement.

Moreover, sometimes reuse might be done with a subset of concepts enlarged with some extra one. 
All this is good supported (with tools) in SKOS context. For OWL you are on your own with your favorite text editor. 

SKOS also organizes your concepts in hierarchies, which you can use for navigation and other purposes. All this is not present in OWL, and must be encoded in OWL if you like it. 

But if your end goal is an OWL ontology you have to do as ontologist the work to combine the two in one model, for instance as I showed in the example above.

Simply put: SKOS is how people manage the codelist/talk about the codelist; while OWL is about how the code list is being used in field.  Two different objectives.

**Comment by bertvannuffelen** [comment]
SKOS is being used as the default.
